### ISSUE TITLE
Fix accessibility issues on search results

### ISSUE THREAD

**Issue by sshaferUCLA** [UCLALibrary/californica#348]
As a visually disabled researcher, I depend on a screen reader to relay the content of an image.

In order to meet that need, I require:
- [x] <alt> text that relays the image content

Currently, this is not provided on a search results page with thumbnails.
Example search results page: https://ursus-stage.library.ucla.edu/catalog?f%5Bsubject_sim%5D%5B%5D=People

- [ ]Thumbnails on this page do not  give the title of the image in the alt tag
EXAMPLE
`<img src="http://californica-stage.library.ucla.edu//downloads/m326m172d?file=thumbnail" alt="M326m172d?file=thumbnail">`

An example of accessible thumbnail images in a search results page, can be found in the database ARTSTOR. 
EXAMPLE
`<img _ngcontent-c14="" aria-role="image" id="LICHTENSTEIN_10313550878" src="//mdxdv.artstor.org/thumb/imgstor/size1/licht/d0001/licht_m-0087_as_8b_srgb.jpg" alt="Thumbnail of Cherry Pie by Lichtenstein, Roy (American painter, sculptor, and printmaker, 1923-1997)" aria-label="Thumbnail of Cherry Pie by Lichtenstein, Roy (American painter, sculptor, and printmaker, 1923-1997)" title="Thumbnail of Cherry Pie by Lichtenstein, Roy (American painter, sculptor, and printmaker, 1923-1997)">`

**Comment by jendiamond** [comment]
Yay, https://groups.google.com/forum/#!topic/blacklight-development/A7bHiIm5QFA

**Comment by jendiamond** [comment]
![Screen Shot 2019-01-30 at 10.11.17 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5615719ec6cad148008cc8ae/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5700b77226457f36b7191a379ef0c2a50415e49bc30254d58eeb9cb74fce36f34985619e8836e0a20c73919a5e98216ee54ba208a1d83bfbb8c4dc5fa2137a5dad11b61435c52dcbd4f3fb2984cc259f34fe94552a51b3f62e570326e0a.png)
